README.md

```diff
@@ -283,12 +311,10 @@ kind: Secret
 metadata:
   name: deis-router-cert
   namespace: deis
-  labels:
-    heritage: deis
```

Comment:
Yes.  Intentionally removed it from the doc because I think it's not relevant here.  This section of the docs would be most applicable to someone endeavoring to use the router in some other context than the Deis PasS, so it seemed an awkward thing to list Deis as the manifest's heritage.